### PR TITLE
Allow to define default network bridge and xen-script

### LINKED ISSUE #22: Automated xl.conf configuration
* Ability to configure xen scripts
   * network
* Option to upload custom xen scripts

### PR DESCRIPTION
**Pull request by Mr-Pi**
This will close #22

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,47 +28,49 @@ Mount or umount the root filesystem from the given VM. For example `systemd star
...
-| `xen_vman_default_auto_install`          | boolean               | `True`                                                              | Install specified OS automatically, when no xen configuration is found                                                                |
+| Name                                     | Type                  |          : Required :         | Default                                                             | Description                                                                                                                           |
+|------------------------------------------|-----------------------|:-----------------------------:|---------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------|
+| `xen_vman_xl_path`                       | string                |               N               | _auto determined_                                                   | Fully qualified path of the xl command                                                                                                |
```
Comment (SlothOfAnarchy) [inline README.md]: Please consider using one column for Required/Default like in the roles written by @dasJ, @Scriptkiddi  and me. I can't force you, but I want to appeal to you and would really like to see it.
I think it's a cleaner way of structuring the variable table, it saves space, gives a better overview and doesn't use too many columns. Besides, in almost all cases, a variable having a default value and a variable not being required occur together, so having two columns for this information is redundant and can be confusing.

File: tasks/configure_xen.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
```
Comment (SlothOfAnarchy) [inline tasks/configure_xen.yml]: Seems fine, but touching the `xl.conf` and #25 definitely belong into the [xen-setup](https://github.com/stuvusIT/xen-setup) role, please commit it there. I don't know how complex the `xl.conf` template would be, but if you plan to do it in the near future, you could just combine both and not need `lineinfile` tasks at all.

Comment (Mr-Pi) [reply tasks/configure_xen.yml]: This was even my first thought ;). But than I realized that xen-setup does a basic hypervisor setup for xen, such things like installing xen, configure grub on a debian based system. xen_vman instead does everything vm management related, such as creating, installing and manage all vms. Creating a proper xl.conf is needed for the tasks done by xen_vman, so it belongs there.

Future more, you can use xen-setup completely unrelated to the tasks done by xen_vman this way. For example you can setup a hypervisor with xen-setup and later use virtsh to manage all vms on this machine, which will conflict with a xl.conf managed by xen-setup.

**Comment by Mr-Pi** [comment]
Please rereview

**Pull request merged**